docs/_includes/xref-interdoc.adoc

```diff
@@ -51,6 +51,7 @@ If [file]_document-b.adoc_ is included in the same master as [file]_document-a.d
 
 Now you can create inter-document cross references without the headache.
 
+[[navigating-between-source-files]]
```

Comment:
We only need explicit IDs if it differs from the auto-generated ID, unless you think the explicit assignment is just good practice.